pandas/_libs/join.pyx

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 
-cimport cython
-from cython cimport Py_ssize_t
+import cython
+from cython import Py_ssize_t
```

Comment:
does this actually matter?

Reply:
No.  Only difference is one is valid python, getting us one step closer to being able to lint